version.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "2.1.14"
```

Comment:
Why?
It's already version 2. We had version 0 for demo and 1 for the first release. After the architecture change we moved to 2. Can you explain your intend here?

Reply:
For testing purposes. I will test pipeline and change back.